### PR TITLE
Website: added nodelist-foreach-polyfill

### PR DESCRIPTION
**Pull request by ianmcburnie**
Replaces `Util.querySelectorAllToArray` with `nodelist-foreach-polyfill`.

### CODE REVIEW

File: docs/ds4.browser.json [hunk 1]
```diff
@@ -3,7 +3,6 @@
         "../src/less/less/ds4/variables.less",
         "../src/less/less/ds4/mixins.less",
         "src/less/ds4/index.less",
-        "./src/js/base/util.js",
```
Comment (seangates) [inline docs/ds4.browser.json]: Where is the new polyfill being included in the bundle?

Comment (ianmcburnie) [reply docs/ds4.browser.json]: Oh! Looks like I may have missed that. Duh! Thanks Sean.

Comment (ianmcburnie) [reply docs/ds4.browser.json]: Although oddly it was working just fine as is in IE11. Still, it should be added to the browser.json.

Comment (ianmcburnie) [reply docs/ds4.browser.json]: Oh, it was picking it up from nested dependency within makeup. Still, will add.

File: docs/ds4.browser.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
...
+        "require-run: nodelist-foreach-polyfill",
```
Comment (seangates) [inline docs/ds4.browser.json]: A nitpick, but could we move this down to where the other polyfill is listed, to keep the requires consistent? Or, does it need to run before the `main.js` files?

Comment (ianmcburnie) [reply docs/ds4.browser.json]: I believe it needs to be before. To be honest, the details one should also be before.

Comment (seangates) [reply docs/ds4.browser.json]: Sounds good. Maybe move the details one up then.

**Pull request merged**